applications/StructuralMechanicsApplication/python_scripts/set_automated_initial_variable_process.py

```diff
@@ -0,0 +1,72 @@
+# Importing the Kratos Library
+from genericpath import isfile
```

Comment:
please use the `pathlib` library, pythons standard way of dealing with any sort of paths